governance/xc_admin/packages/xc_admin_frontend/components/InstructionViews/WormholeInstructionView.tsx

```diff
@@ -408,12 +409,29 @@ export const WormholeInstructionView = ({
                 <CopyText text={'0x' + governanceAction.callAddress} />
               </div>
               <div>Value: {governanceAction.value.toString()}</div>
-              <div>
-                Call Data:{' '}
-                <CopyText
-                  text={'0x' + governanceAction.calldata.toString('hex')}
-                />
-              </div>
+              {(() => {
+                const parsedData = parseEvmExecuteCallData(
+                  governanceAction.calldata.toString('hex')
+                )
+                if (parsedData === undefined) return <></>
+                return (
+                  <>
+                    <div>Call Method: {parsedData.method}</div>{' '}
+                    <div>
+                      Call Params:{' '}
+                      {parsedData.inputs.length > 0 ? (
+                        parsedData.inputs.map(([key, value]) => (
+                          <div key={key} className="mx-4">
+                            {key}: {value as string}
```

Comment:
we should do this cast on the parsing side if necessary and not propagate the `unknown` type up to here